### PR TITLE
Zooms Today: Quickly launch the Zoom link attached to meetings in your calendar

### PR DESCRIPTION
**Pull request by sprak3000**
This plug-in scans the current day in your calendar to find events containing a Zoom link. If any are found, they are presented in a list where clicking an item will open the link to that Zoom meeting.

### CODE REVIEW

File: Lifestyle/ZoomsToday/zooms-today.1h.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#
+# Display a list of Zoom based meetings in your calendar today.
+# Selecting a meeting will take you to the Zoom session associated with it.
+#
+# <bitbar.title>Zoom Meetings for Today</bitbar.title>
+# <bitbar.version>1.0.0</bitbar.version>
+# <bitbar.author>Luis Cruz</bitbar.author>
+# <bitbar.author.github>sprak3000</bitbar.author.github>
+# <bitbar.desc>Display a list of Zoom based meetings in your calendar today. Selecting a meeting will take you to the Zoom session associated with it.</bitbar.desc>
+# <bitbar.dependencies>icalBuddy</bitbar.dependencies>
```
Comment (matryer) [inline Lifestyle/ZoomsToday/zooms-today.1h.sh:12]: Would it be possible to get an image here?

**Comment by matryer** [comment]
This is very cool!

File: Lifestyle/ZoomsToday/zooms-today.1h.sh (hunk 1 above)
Comment (sprak3000) [reply Lifestyle/ZoomsToday/zooms-today.1h.sh:13]: Done!

**Comment by sprak3000** [comment]
> This is very cool!

It's even cooler you have made something like this possible. Thank you for that!

**Comment by matryer** [comment]
Nice one @sprak3000

**Pull request merged**